wix.json

```diff
@@ -0,0 +1,22 @@
+{
+  "product": "Harness-Docker-Runner",
+  "company": "Harness",
+  "license": "LICENSE.md",
+  "upgrade-code": "",
+  "files": {
+    "guid": "",
+    "items": [
+      "./lite-engine.exe"
+    ]
+  },
+  "directories": [
+    "templates"
+  ],
+  "hooks": [
+    {"when": "install", "command": "sc.exe create harness-docker-runner-svc binPath=\"[INSTALLDIR]lite-engine.exe server\" start=auto DisplayName=\"harness-docker-runner-svc\""},
+    {"when": "install", "command": "sc.exe start harness-docker-runner-svc"},
+    {"when": "uninstall", "command": "sc.exe query harness-docker-runner-svc"},
```

Comment:
instead of using inline command do, {"when": "uninstall", "batch-file": "uninstall.bat"}